pipelines/rhtap-service-push/rhtap-service-push.yaml

```diff
@@ -293,21 +319,21 @@ spec:
       runAfter:
         - push-sbom-to-pyxis
   finally:
-    - name: slack-webhook-notification
+    - name: send-slack-notification
       taskRef:
         resolver: "git"
         params:
           - name: url
-            value: https://github.com/redhat-appstudio/release-service-catalog.git
+            value: https://github.com/scoheb/release-service-catalog.git
           - name: revision
-            value: main
+            value: refactor-slack
           - name: pathInRepo
-            value: tasks/slack-webhook-notification/slack-webhook-notification.yaml
+            value: tasks/send-slack-notification/send-slack-notification.yaml
       params:
         - name: message
-          value: Release pipelineRun $(context.pipelineRun.name) $(tasks.status)
-        - name: dataJsonPath
-          value: "$(workspaces.data.path)/$(context.pipelineRun.uid)/data.json"
+          value: $(tasks.collect-slack-notification-params.results.message)
+        - name: tasksStatus
+          value: "$(tasks.status)"
```

Comment:
I've never seen this before. What does it do?

Reply:
See https://tekton.dev/docs/pipelines/variables/ ... search for `tasks.status`

Reply:
> An aggregate status of all the pipelineTasks under the tasks section (excluding the finally section). This variable is only available in the finally tasks and can have any one of the values (Succeeded, Failed, Completed, or None)

Reply:
added comment